code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -169,6 +173,7 @@
 	key = "gnarl"
 	key_third_person = "gnarls"
 	message = "gnarls and shows its teeth..."
+	message_mime = "gnarls silently, baring its teeth..."
```

Comment:
I really think "their" > "its" in this case, but whatever the normal message is making the mistake too

Reply:
I would agree too, but in this context it's a monkey doing this emote, seeing as monkeys arent't usually sentient, its makes more sense, I just added this kind of as a joke and kind of seriously, nobody ever considers the monkified mimes...

Reply:
shouldn't this use one of the pronoun helpers or something

Reply:
Again, it is a monkey emote, a monkey does this emote, I don't think a humanoid can, so for consistencies sake I made the pronoun "its" to match with aaaaall the other monkey emotes, hell, even the normal emote _right_ above the mime version uses its, but if you insist I will change it